WordPress/Classes/Utility/Logging/EventLoggingDataProvider.swift

```diff
@@ -10,20 +10,23 @@ struct EventLoggingDataProvider: EventLoggingDataSource {
     /// A block that returns all existing log files
     private let fetchLogFiles: LogFilesCallback?
 
+    let loggingAuthenticationToken: String
+    let loggingEncryptionKey: String
```

Comment:
I removed the documentation comments when moving these two properties above the init because they are inherited from `EventLoggingDataSource`.

---

P.S. I'm surprised the verb to use is "inherited" when the property are defined in the protocol, I would have used something like "required", but that's what Xcode uses:

![image](https://github.com/user-attachments/assets/d1193f86-e4b0-48a0-a146-c137d64b78ed)